### DISCUSSION TITLE
Deadlock situation in registration

### DISCUSSION THREAD

**Discussion by smartqrdev** [vercel/community#3995] [Account Issue] [answered]
### Question

I registered on Vercel using my email address and successfully verified it by clicking the email verification link. However, when I tried to log in, the platform asked for my phone number and indicated that my country, Sri Lanka, is not eligible for registration. To work around this, I attempted to sign up using the GitHub option, but I received a message stating that the email address is already registered. As a result, I'm currently unable to log in or sign up using my GitHub account.

**Comment by swarnava** [comment, marked as answer]
Hi,

Please email us at registration@vercel.com
